HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1243,135 +1250,154 @@ def self.apply_local_ventilation(model, vent_object_array, obj_type_name)
     return obj_sch_sensors
   end
 
-  def self.calc_hrv_erv_effectiveness(vent_mech)
-    vent_mech_cfm = vent_mech.average_flow_rate
-    if (vent_mech_cfm > 0)
-      # Must assume an operating condition (HVI seems to use CSA 439)
-      t_sup_in = 0.0
-      w_sup_in = 0.0028
-      t_exh_in = 22.0
-      w_exh_in = 0.0065
-      cp_a = 1006.0
-      p_fan = vent_mech.average_fan_power # Watts
+  def self.calc_hrv_erv_effectiveness(vent_mech_fans)
+    # Create the mapping between mech vent instance and the effectiveness results
+    hrv_erv_effectiveness_map = {}
```

Comment:
Change this method to return a map, so that it can only be called once at the top level and store data for all methods related to HRV/ERV calculations.